app/communication/models/banner.py

```diff
@@ -0,0 +1,57 @@
+import uuid
+
+from django.db import models
+from django.db.models import Q
+from django.utils import timezone
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.util.models import BaseModel, OptionalImage
+from app.util.utils import now
+
+
+def tomorrow():
+    return timezone.now() + timezone.timedelta(days=1)
+
+
+class Banner(BaseModel, OptionalImage, BasePermissionModel):
+    write_access = AdminGroup.admin()
+
+    id = models.UUIDField(
+        auto_created=True,
+        primary_key=True,
+        default=uuid.uuid4,
+        serialize=False,
+    )
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    visible_from = models.DateTimeField(default=timezone.now)
+    visible_until = models.DateTimeField(default=tomorrow)
+    url = models.URLField(max_length=600, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.title} {'- is visible' if self.is_visible else ''} - {self.description}"
+
+    def save(self, *args, **kwargs):
+        if self.is_visible and self.is_uniquely_visible:
+            raise ValueError("Det finnes allerede et banner som er synlig")
```

Comment:
Mener `ValueError` vil returnere HTTP 500 i responsen siden det er et unntak innebygd i Python. Kan du dobbeltsjekke og evt gjøre noe liknende som i `RegistrationViewSet` hvor vi bruker en [mixin](https://github.com/TIHLDE/Lepton/blob/85388641627cd8ac352d88fe21aef5339488ea7d/app/content/mixins.py#L14) for å mappe unntak til riktig statuskode og detail-melding?

Reply:
Jeg kan ta en nærmere titt. Tanken bak var det som er gjort i weekly business.